package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "badaso-lms-theme",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "npx jest"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.17.5",
+    "@babel/preset-env": "^7.16.11",
+    "@vue/compiler-dom": "^3.2.31",
+    "@vue/test-utils": "^1.3.0",
+    "@vue/vue2-jest": "^27.0.0-alpha.4",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^27.5.1",
+    "jsdom-global": "^3.0.2"
+  },
+  "jest": {
+    "moduleFileExtensions": [
+      "js",
+      "json",
+      "vue"
+    ],
+    "testURL": "http://localhost/",
```

Comment:
with this value, is it runnable on github workflow?

Reply:
well it runs on pipeline for now